WebColock.Tests/IntegrationTests/IntegrationClockInOutControllerTests.cs

```diff
@@ -0,0 +1,47 @@
+﻿using FluentAssertions;
+using WebClock.Controllers.Dtos;
+using Xunit;
+namespace WebColock.Tests.IntegrationTest
+{
+    public class IntegrationClockInOutControllerTests
+    {
+        private readonly ServerSut _server = new ServerSut();
+        const int _userId = 1;
+
+        [Fact]
+        public async void ClockIn_Test()
+        {
+            await _server
+                .DoPost<object>($"api/ClockInOut/ClockIn/{_userId}", new object());
+            //todo usunąć <object>
+        }
+
+        [Fact]
+        public async void ClockOut_Test()
+        {
+            var server = new ServerSut();
+            var userId = 1;
```

Comment:
to może być const

Reply:
A nawet można użyć pola _server oraz stałej UserId.